modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -67,16 +68,11 @@
  */
 public class AssignmentsImpl implements Assignments {
 
+    protected static final String RULE_CACHE = "ruleCache";
```

Comment:
@jcwuzoegiver  well as a part of u/g autumn i did not expect to see the included AssignmentsImpl mods ... wonder if i picked them up from somewhere else?